custom_search_commands/python/eventingsearchcommands_app/default/app.conf

```diff
@@ -13,3 +13,5 @@ label = Eventing CSC
 description = Eventing custom search commands example
 version = 1.0.0
 
+[package]
+check_for_updates = false
```

Comment:
Appinspect error?

Reply:
yes, this change is required to pass appinspect check, so Splunk Enterprise skips checking Splunkbase for updates to this app.